modules/fundamental/src/vulnerability/service/mod.rs

```diff
@@ -35,100 +27,25 @@ impl VulnerabilityService {
 
     pub async fn fetch_vulnerabilities<C: ConnectionTrait + Sync + Send>(
         &self,
-        search: Query,
+        mut search: Query,
         paginated: Paginated,
         deprecation: Deprecation,
         connection: &C,
     ) -> Result<PaginatedResults<VulnerabilitySummary>, Error> {
-        let inner_query = vulnerability::Entity::find()
-            .left_join(cvss3::Entity)
-            .expr_as_(
-                SimpleExpr::FunctionCall(Func::avg(SimpleExpr::Column(
-                    trustify_entity::cvss3::Column::Score.into_column_ref(),
-                ))),
-                "average_score",
-            )
-            .expr_as_(
-                SimpleExpr::FunctionCall(Func::cust("cvss3_severity".into_identity()).arg(
-                    SimpleExpr::FunctionCall(Func::avg(SimpleExpr::Column(
-                        trustify_entity::cvss3::Column::Score.into_column_ref(),
-                    ))),
-                )),
-                "average_severity",
-            )
-            .group_by(vulnerability::Column::Id);
-
-        let mut outer_query = vulnerability::Entity::find();
+        search.q = search.q.replace("average_severity", "base_severity");
```

Comment:
This is a temporary workaround, until we update UI to use new values.
@jcrossley3 I tried playing with the aliases, but failed miserably. If you have any idea on how it could be done, please let me know. Although, this should be removed altogether soon, so I wouldn't spend too much time on it.

Reply:
OK, removed the "aliasing" and created UI PR (https://github.com/trustification/trustify-ui/pull/499).

It is definitely a breaking change, but I didn't see those notes used in trustify project. Any pointers on how it should look?

On the other note, this is just one change in the whole set that will change the API and need data reimport. So we'll need custom release notes once it's done.

Reply:
I found the BREAKING_CHANGE example and added a description for this commit. Thanks for the pointer.